superset/databases/api.py

```diff
@@ -909,11 +908,15 @@ def available(self) -> Response:
                 "preferred": engine_spec.engine in preferred_databases,
             }
 
-            if issubclass(engine_spec, BasicParametersMixin):
-                payload["parameters"] = engine_spec.parameters_json_schema()
+            if hasattr(engine_spec, "parameters_json_schema") or hasattr(
```

Comment:
Just realized this needs to be `and`, otherwise we might get an `AttributeError`:

```suggestion
            if hasattr(engine_spec, "parameters_json_schema") and hasattr(
```